process.rb

```diff
@@ -180,6 +172,17 @@ def slugify(word)
     f.puts('---')
   end
 end
+OaklandCandidate.find_each do |committee|
+  build_file("/_committees/#{committee.FPPC}.md") do |f|
+    f.puts(YAML.dump(
+      'filer_id' => committee.FPPC.to_s,
+      'name' => committee.Committee_Name,
+      'candidate_controlled_id' => '',
+      'title' => committee.Committee_Name
+    ))
+    f.puts('---')
+  end
+end
```

Comment:
Mostly looks good, but this results in some candidate_controlled_id values disappearing.

```diff
diff --git a/build/_committees/1381183.md b/build/_committees/1381183.md
index 90bb88a..ca00fe4 100644
--- a/build/_committees/1381183.md
+++ b/build/_committees/1381183.md
@@ -1,6 +1,6 @@
 ---
 filer_id: '1381183'
 name: Kaplan for Oakland City Council 2016
-candidate_controlled_id: '1406223'
+candidate_controlled_id: ''
 title: Kaplan for Oakland City Council 2016
 ---
```

Reply:
I noticed that at one point and was going to look into it... the problem is that this is an old candidate committee.  I would think it would have been terminated by this time.  I think we can ignore for now as Kaplan is no longer running in this election so the information will not be displayed anywhere.